packages/thumbprint-react/tsconfig.types.json

```diff
@@ -0,0 +1,20 @@
+// TS config for generate type delcaration files only. This does not produce any JS code.
+// These .d.ts files are included in the dist directory along with the Rollup output, so that
+// types are available to consumers of the library.
+{
+    "extends": "../../tsconfig.json",
+    "compilerOptions": {
+        // Default to generating types for the ES Modules, we also override this to produce types
+        // for the CJS modules in the `generateTypeDeclarations` script in `package.json`.
+        "outDir": "./dist/es",
+        "rootDir": ".",
+        "declaration": true,
+        "declarationMap": true,
+        "isolatedModules": false,
+        "noEmit": false,
+        "allowJs": false,
+        "emitDeclarationOnly": true
+    },
+    // TODO(giles): figure out how to include more than this one file here.
+    "files": ["components/ButtonRow/index.tsx"]
```

Comment:
If this were `index.js` instead, would TypeScript know to follow the imports and create the type definitions for `ButtonRow`?

Reply:
I tried that and it didn't seem to work.

The correct way to get all the TS files here would be to do `"includes": ["**/*.{ts,tsx}"]` or similar, since `includes` takes a glob and `files` doesn't. But for some reason when I specified that it wasn't finding any files. Maybe my glob was wrong.